pkg/webhook/validate_service.go

```diff
@@ -28,9 +28,18 @@ import (
 	v1 "knative.dev/serving/pkg/apis/serving/v1"
 )
 
-var (
-	// PodSpecDryRunAnnotation gates the podspec dryrun feature and runs with the value 'enabled'
-	PodSpecDryRunAnnotation = "features.knative.dev/podspec-dryrun"
+// PodSpecDryRunAnnotation gates the podspec dryrun feature and runs with the value 'enabled'
+var PodSpecDryRunAnnotation = "features.knative.dev/podspec-dryrun"
+
+// DryRunMode represents possible values of the PodSpecDryRunAnnotation
+type DryRunMode string
+
+const (
+	// Enabled will run the dryrun logic. Will succeed if dryrun is unsupported.
+	Enabled DryRunMode = "enabled"
```

Comment:
this ends up being `webhook.Enabled` outside the package, maybe better as `webhook.DryRunEnabled` or something?

Reply:
+1 - same with `Strict` below